app/models/external_registries/external_registries.rb

```diff
@@ -0,0 +1,21 @@
+module ExternalRegistries
+  module ExternalRegistries
```

Comment:
why the double module?

Reply:
Following naming conventions subject to the constraint of `ExternalRegistries` not living in `/app/models`. Constraint relaxed and namespaces flattened in af5a42df